docs/install.rst

```diff
@@ -0,0 +1,64 @@
+Install
+=======
+
+Below we assume you have the default Python environment already configured on
+your computer and you intend to install ``rerf`` inside of it.  If you want
+to create and work with Python virtual environments, please follow instructions
+on `venv <https://docs.python.org/3/library/venv.html>`_ and `virtual
+environments <http://docs.python-guide.org/en/latest/dev/virtualenvs/>`_.
+
+First, make sure you have the latest version of ``pip`` (the Python package manager)
+installed. If you do not, refer to the `Pip documentation
+<https://pip.pypa.io/en/stable/installing/>`_ and install ``pip`` first.
+
+Requirements
+------------
+
+Python bindings use pybind11_.
+
+.. _pybind11: https://github.com/pybind/pybind11
+
+C++ compiler (gcc)::
+
+  sudo apt-get install build-essential cmake python3-dev libomp-dev   # Ubuntu/Debian
+  
+
+On Mac:
+
+- Get `brew`_
+- ``brew install python``
```

Comment:
Do we need to specify python3 here?  I'm not sure what the default is for `brew`.